packages/mattermost-redux/src/actions/bots.test.ts

```diff
@@ -37,7 +35,7 @@ describe('Actions.Bots', () => {
 
         const state = store.getState();
         const botsResult = state.entities.bots.accounts;
-        assert.equal(bots.length, Object.values(botsResult).length);
```

Comment:
nothing to change here, i just noticed that 
- nodejs asset equal does shallow and we have changed them to deep equal
- in places which were doing `deepEqual` we changed them to `equal` instead of [strict equal](https://jestjs.io/docs/expect#tostrictequalvalue)